scripts/install-worker.sh

```diff
@@ -111,7 +111,7 @@ sudo yum install -y yum-utils device-mapper-persistent-data lvm2
 INSTALL_DOCKER="${INSTALL_DOCKER:-true}"
 if [[ "$INSTALL_DOCKER" == "true" ]]; then
     sudo amazon-linux-extras enable docker
-    sudo groupadd -fog 1950 docker && sudo useradd --gid 1950 docker
+    sudo groupadd -f docker
```

Comment:
To avoid backwards incompatibility issues in the event that someone depends on this, could you just update it to something like this?

```
sudo groupadd -fog 1950 docker && sudo useradd --gid $(id -g docker) docker
```

Reply:
Looks like `groupadd` is a no-op if the group already exists. Also, that `id` command work for me, you'll probably need to do something else.

```
$ sudo groupadd -fog 1999 docker
$ sudo groupadd -fog 1950 docker
$ cut -d: -f3 < <(getent group homey)
> 1999
```